wolfssl/wolfcrypt/types.h

```diff
@@ -761,13 +761,12 @@ decouple library dependencies with standard string, memory and so on.
         #define XSTRLCAT(s1,s2,n) wc_strlcat((s1),(s2),(n))
     #endif
 
-    #if !defined(NO_FILESYSTEM) && defined(OPENSSL_EXTRA) && \
-        !defined(NO_STDIO_FILESYSTEM)
+    #if !defined(NO_FILESYSTEM) && !defined(NO_STDIO_FILESYSTEM)
```

Comment:
Worried about the stdlib.h for embedded customers. Can you include `HAVE_STDLIB_H` or do you think `NO_STDIO_FILESYSTEM` will be good enough?

Reply:
I did some more research on this.  we have a slew of other `#include <stdlib.h>`s around and about, gated on various things like particular ports, `!defined(XATOI)`, `defined(WOLFSSL_MEMORY_LOG)`, `defined(NO_WOLFSSL_MEMORY) && !defined(WOLFSSL_NO_MALLOC)`, and `!defined(STRING_USER) && !defined(WOLFSSL_LINUXKM)`.

we're not using `HAVE_STDLIB_H` anywhere in the code -- not sure it's even getting detected by or exported from autotools.